#169674195 Fix failing coverage script on Travis

RaymondMwaura opened the pull request:
#### What does this PR do?
This PR fixes a bug with the Coveralls and Travis configuration that results in test coverage not being published on Coveralls.io

#### Description of Task to be completed?
 - [x] update the 'travis.yml' file
 - [x] update the 'package.json' file
 - [x] update the 'README.md' file

#### How should this be manually tested?
-Push a branch to this repository
-After the tests have passed on TravisCI, log into Coveralls.io with you Github account to view the test coverage of the branch

#### Any background context you want to provide?
This PR supplements the PR that created test coverage, that is [#169257390](https://www.pivotaltracker.com/story/show/169257390)

#### What are the relevant pivotal tracker stories?
[#169674195](https://www.pivotaltracker.com/story/show/169674195)

#### Screenshots (if appropriate)
<img width="1437" alt="Coverage Fixed" src="https://user-images.githubusercontent.com/17108099/68636219-74bff780-0503-11ea-8af5-b79b32c90184.png">

#### Questions:
N/A

b0nbon1 commented on package.json:
```diff
@@ -11,7 +11,7 @@
   },
   "scripts": {
-    "test": "set NODE_ENV=test && nyc mocha src/tests/*.js --require @babel/register",
+    "test": "npm run build && nyc mocha dist/tests/*.js",
```
nice work but you have to set the environments to test so  add the testing environment to the script

RaymondMwaura committed:
fix(base-config): fix failing travis coveralls coverage

-fix failing coverage on Travis to get Coveralls working properly

[Delivers ##169674195]
